src/utils/feeBreakdown.ts

```diff
@@ -98,44 +105,55 @@ const getReductionOrExemptionLists = (data: PassportFeeFields) => {
   amount: {
     calculated: getCalculatedAmount(data),
+    calculatedVAT: data["application.fee.calculated.VAT"],
     payable: data["application.fee.payable"],
-    vat: calculateVAT(data),
+    payableVAT: data["application.fee.payable.VAT"],
+    serviceCharge: data["application.fee.serviceCharge"],
+    serviceChargeVAT: data["application.fee.serviceCharge.VAT"],
+    fastTrack: data["application.fee.fastTrack"],
+    fastTrackVAT: data["application.fee.fastTrack.VAT"],
+    paymentProcessing: data["application.fee.paymentProcessing"],
+    paymentProcessingVAT: data["application.fee.paymentProcessing.VAT"],
     ...calculateReductionOrExemptionAmounts(data),
   },
   ...getReductionOrExemptionLists(data),
 });
 
-export const createPassportSchema = () => {
-  const questionSchema = z.number().nonnegative();
-  const setValueSchema = z.tuple([z.coerce.number().nonnegative()]);
-  const feeSchema = z
-    .union([questionSchema, setValueSchema])
-    .transform(toNumber);
-
-  /** Describes how boolean values are set via PlanX components */
-  const booleanSchema = z
-    .tuple([z.enum(["true", "false"])])
-    .default(["false"])
-    .transform(toBoolean);
-
-  const schema = z
-    .object({
-      "application.fee.calculated": feeSchema.optional().default(0),
-      "application.fee.payable": feeSchema,
-      "application.fee.payable.includesVAT": booleanSchema,
-      "application.fee.reduction.alternative": booleanSchema,
-      "application.fee.reduction.parishCouncil": booleanSchema,
-      "application.fee.reduction.sports": booleanSchema,
-      "application.fee.exemption.disability": booleanSchema,
-      "application.fee.exemption.resubmission": booleanSchema,
-    })
-    .transform(toFeeBreakdown);
-
-  return schema;
-};
+const questionSchema = z.number().nonnegative();
+const setValueSchema = z.tuple([z.coerce.number().nonnegative()]);
+const feeSchema = z.union([questionSchema, setValueSchema]).transform(toNumber);
+
+/** Describes how boolean values are set via PlanX components */
+const booleanSchema = z
+  .tuple([z.enum(["true", "false"])])
+  .default(["false"])
+  .transform(toBoolean);
+
+/** Static fee-associated passport fields */
+export const staticSchema = z.object({
+  "application.fee.calculated": feeSchema.optional().default(0),
+  "application.fee.calculated.VAT": feeSchema.optional().default(0),
+  "application.fee.payable": feeSchema, // only required fee and inclusive of VAT
+  "application.fee.payable.VAT": feeSchema.optional().default(0), // sum of all VAT
+  "application.fee.fastTrack": feeSchema.optional().default(0),
+  "application.fee.fastTrack.VAT": feeSchema.optional().default(0),
+  "application.fee.serviceCharge": feeSchema.optional().default(0),
+  "application.fee.serviceCharge.VAT": feeSchema.optional().default(0),
+  "application.fee.paymentProcessing": feeSchema.optional().default(0),
+  "application.fee.paymentProcessing.VAT": feeSchema.optional().default(0),
```

Comment:
Given the number of `.optional().default()` here vs the single required fee, it might be clearer to capture these as variables?

```ts
const optinalFeeSchema = z.union([questionSchema, setValueSchema]).transform(toNumber).optional().default(0);
const requiredFeeSchema = z.union([questionSchema, setValueSchema]).transform(toNumber);
```